generated/proto/namespace/namespace.proto

```diff
@@ -11,13 +11,15 @@ message RetentionOptions {
 }
 
 message NamespaceOptions {
-	bool needsBootstrap      = 1;
-	bool needsFlush          = 2;
-	bool writesToCommitLog   = 3;
-	bool needsFilesetCleanup = 4;
-	bool needsRepair         = 5;
+	bool bootstrapEnabled      	= 1;
```

Comment:
super nit: still seeing whitespace issues in the PR view. looks like you're using tabs to align, could you switch to spaces. simple way to see this - open the file in vim, and do `:set list` to visualize whitespace characters.

Reply:
ah damn, ok I'll try that but I think there is a "convert all tabs to spaces" command in VScode I can just run

Reply:
done